starklings-backend/db.py

```diff
@@ -0,0 +1,36 @@
+import os
+import pymysql
+from dotenv import load_dotenv
+load_dotenv()
+
+connection = pymysql.connect(
```

Comment:
It would be cleaner to use migrations, maybe you can create an issue to work on this?

Reply:
Yeap, what about getting  a v 1 with this and I 'll update a new issue

Reply:
Sure